### PR TITLE
refactor: date buttons to be glimmer component

### PR DESCRIPTION
**Pull request by MitanOmar**

### CODE REVIEW

**Comment by derrabauke** [comment]
You have to relocate the `.scss` file, co-location is not supported with glimmer components.

File: app/components/date-buttons/component.js [hunk 1]
```diff
@@ -1,48 +1,47 @@
-import Component from "@ember/component";
 import { action } from "@ember/object";
-import classic from "ember-classic-decorator";
+import Component from "@glimmer/component";
 import moment from "moment";
 
-@classic
 export default class DateButtons extends Component {
```
Comment (derrabauke) [inline app/components/date-buttons/component.js:5]:
```suggestion
export default class DateButtonsComponent extends Component {
```

**Comment by derrabauke** [comment]
**Before:**
image

**After:**
image

This is caused by the lack of the `.scss` file.

Please check the stuff you touch also visually. Such regressions are annoying for the users and lead to a lot of lost time, but don't require much time while refactoring. ;)

File: app/components/date-buttons/component.js [hunk 2]
```diff
@@ -1,48 +1,47 @@
...
-    this.set("choices", [
-      "this week",
-      "this month",
-      "this year",
-      "last week",
-      "last month",
-      "last year",
-    ]);
-  }
+export default class DateButtonComponent extends Component {
```
Comment (derrabauke) [inline app/components/date-buttons/component.js:5]: Sorry for the nitpicking :sweat_smile: 

```suggestion
export default class DateButtonsComponent extends Component {
```

File: app/styles/app.scss [hunk 3]
```diff
@@ -43,6 +43,7 @@
 @import "components/sy-toggle";
 @import "components/progress-tooltip";
 @import "components/welcome-modal";
+@import "components/date-button";
```
Comment (derrabauke) [inline app/styles/app.scss:46]:
```suggestion
@import "components/date-buttons";
```

File: app/styles/components/date-button.scss
Comment (derrabauke) [inline app/styles/components/date-button.scss:1]: `app/styles/components/date-button.scss`

File: app/components/date-buttons/component.js (hunk 2 above)
Comment (derrabauke) [reply app/components/date-buttons/component.js:5]: :point_up_2:

Comment (MitanOmar) [reply app/components/date-buttons/component.js:5]: lool, changing

**Review by derrabauke — APPROVED** [review]
:clap:

**Pull request merged**